### PR TITLE
BaSP-TG-36: admin list

### PR DESCRIPTION
**Pull request by nicomicheletti**

### CODE REVIEW

**Review by FranGutierrez — APPROVED** [review]
Looks and works correctly!

**Review by MatiasJoaquinReynoso — CHANGES_REQUESTED** [review]
Hello Nico, reviewing your pr I can say that the CRUD methods works correctly !!, I only make a few observations.

1- When creating admin there is no redirection (if I press ok in the modal everything is kept on the same page) and if I continue pressing the save button with the same validations the same user is created many times

2- When deleting an id it does not send me a modal for confirmation of delete from user

**Review by MatiasJoaquinReynoso — APPROVED** [review]

**Review by Julian25 — APPROVED** [review]

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,56 +1,120 @@
...
 
-  const createAdmin = (adm) => {
-    const newAdmin = { ...adm };
-    setAdmins([...admins, newAdmin]);
+  const getAdmins = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins`);
+      const body = await response.json();
+      setAdmins(body.data);
+      setIsLoading(false);
```
Comment (BaltazarCortina1) [inline src/Components/Admins/index.js:61]: Sigue faltando validar la respuesta.

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,56 +1,120 @@
...
       <div>
-        <button onMouseDown={() => changeScreen(false)}>Admin List</button>
-        <button onMouseDown={() => changeScreen(true)}>Save Admin</button>
+        <Link
+          to={{
+            pathname: '/admins/form',
+            linkData: config,
+            DBPath: serverPath
+          }}
+          className={styles.LinkReset}
```
Comment (BaltazarCortina1) [inline src/Components/Admins/index.js:105]: Esto ya no es necesario, solo la ruta

File: src/Components/Admins/index.js [hunk 3]
```diff
@@ -1,56 +1,120 @@
...
+import Loading from '../Shared/Loading/Loading';
 
 const Admins = () => {
   const [admins, setAdmins] = useState([]);
-  const [screen, changeScreen] = useState(false);
-  const [method, setMethod] = useState('POST');
-  const [adminId, setAdminId] = useState('');
+  const [isLoading, setIsLoading] = useState([true]);
+  const serverPath = '/admins';
+  const config = [
```
Comment (BaltazarCortina1) [inline src/Components/Admins/index.js:12]: Esto ya no es necesario aca

**Pull request merged**